packages/client/hmi-client/src/components/model/petrinet/tera-distribution-input.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+	<span class="flex gap-2">
+		<section>
+			<Dropdown
+				:id="parameterId"
+				ref="dropdownRef"
+				:model-value="getParameterDistribution(modelConfiguration, parameterId).type"
+				@change="onChange"
+				@hide="hidePopup"
+				option-label="name"
+				option-value="value"
+				:options="distributionOptions"
+				class="mr-3 parameter-input"
+			>
+				<template #option="slotProps">
+					<section
+						class="flex align-items-center w-full"
+						@mouseover="showPopup(dropdownRef.$el, slotProps.index)"
+						@focusin="() => showPopup(dropdownRef.$el, slotProps.index)"
+					>
+						<div>{{ slotProps.option.name }}</div>
+					</section>
+				</template>
+			</Dropdown>
+			<section class="popup" :style="{ top: `${popupPosition.top}px`, left: `${popupPosition.left}px` }">
+				<Card v-if="hoveredOption" ref="cardRef" class="inside p-2">
+					<template #title>
+						{{ hoveredOption?.name }}
+						<a
+							v-if="distributions[hoveredOption.value].url"
+							v-tooltip="hoveredOption.name + ' pytorch.org docs'"
+							:href="distributions[hoveredOption.value].url"
+							:aria-label="hoveredOption.name + ' docs'"
+							rel="noopener noreferrer"
+							target="_blank"
+						>
+							<span class="pi pi-external-link"></span>
+						</a>
+					</template>
+					<template #content>
+						{{ distributions[hoveredOption?.value].description }}
+						<h5 class="pt-3 pb-2">Required parameters:</h5>
+						<ul class="ml-5">
+							<li class="pb-2" v-for="param in distributions[hoveredOption.value].parameters" :key="param">
+								<h6>{{ param.label }}</h6>
+								<p class="parameter-description">
+									{{ param.description }}
+								</p>
+							</li>
+						</ul>
+					</template>
```

Comment:
I think you are overcomplicating things here. Just make a massive v-if list of tootip hard coded. Using the JSON to do that is source of confusion and hard to update compare to a simple HTML element. Let's chat if you want more info

Reply:
change made